configs/services/dynamodb.json

```diff
@@ -1,3 +1,21 @@
 {
-    "libraryName": "amazonka-dynamodb"
+    "libraryName": "amazonka-dynamodb",
+    "extraDependencies": [
+        "aeson",
+        "containers",
+        "hashable >=1.3.4.0 && <1.5",
```

Comment:
We need `hashable >= 1.3.4.0` to get `instance (Hashable k, Hashable v) => Hashable (Map k v)`. I am happy to raise the bound on `amazonka-core` instead if you prefer.

Reply:
I'm fine with the constraint being here